### ISSUE TITLE
Docker Image build fails on Windows 10 : failed to solve with frontend dockerfile.v0: failed to solve with frontend gateway.v0: rpc error: code = Unknown desc = failed to build LLB: executor failed running

### ISSUE THREAD

**Issue by MeherAyedi** [oss-review-toolkit/ort#3263]
Cloning the code on Windows 10 machine and Docker v19.03.13 and then running : "docker build -t ort ." will throw an error:

...
...
...
#8 38.68 Fetched 815 B in 1s (955 B/s)
#8 38.68 Reading package lists...
#8 40.26 Reading package lists...
#8 41.79 Building dependency tree...
#8 42.09 Reading state information...
#8 42.29 E: Version '9.0.1-2.3~ubuntu1.18.04.3' for 'python-pip' was not found
#8 42.29 E: Version '9.0.1-2.3~ubuntu1.18.04.3' for 'python3-pip' was not found
------
failed to solve with frontend dockerfile.v0: failed to solve with frontend gateway.v0: rpc error: code = Unknown desc = failed to build LLB: executor failed running [/bin/sh -c apt-get update &&     apt-get install -y --no-install-recommends gnupg &&     echo 'Acquire::https::dl.bintray.com::Verify-Peer "false";' | tee -a /etc/apt/apt.conf.d/00sbt &&     echo "deb https://dl.bintray.com/sbt/debian /" | tee -a /etc/apt/sources.list.d/sbt.list &&     curl -ksS "https://keyserver.ubuntu.com/pks/lookup?op=get&options=mr&search=0x2EE0EA64E40A89B84B2DF73499E82A75642AC823" | apt-key adv --import - &&     curl -sL https://deb.nodesource.com/setup_12.x | bash - &&     apt-get update &&     apt-get install -y --no-install-recommends         lib32stdc++6         libffi-dev         libgmp-dev         libxext6         libxi6         libxrender1         libxtst6         make         netbase         openssh-client         unzip         xz-utils         zlib1g-dev         cvs         git         mercurial         subversion         bundler=$BUNDLER_VERSION         cargo=$CARGO_VERSION         composer=$COMPOSER_VERSION         nodejs         python-pip=$PYTHON_PIP_VERSION         python-setuptools         python3-pip=$PYTHON_PIP_VERSION         python3-setuptools         sbt=$SBT_VERSION     &&     rm -rf /var/lib/apt/lists/*]: runc did not terminate sucessfully
************************

Any ideas why this is happening?

**Comment by sschuberth** [comment]
There are [reports](https://stackoverflow.com/a/59868693/1127485) that giving Docker more memory helps...

**Comment by sschuberth** [comment]
...or maybe also [more disk space](https://github.community/t/worker-running-out-of-disk-space/18245).

**Comment by MeherAyedi** [comment]
Tried both to increase memory, CPU and Disk image, though it doesn't work.

**Comment by sschuberth** [comment]
Is the issue still reproducible?

**Comment by MeherAyedi** [comment]
The issue still reproducible

**Comment by sschuberth** [comment]
I just tried again with latest `master`, and for me the Docker image built fine on Windows (and on Linux anyway). So I suspect it's something about your setup.

The only issue I came across was that Docker was throwing an [Unexpected EOF](https://github.com/docker/for-win/issues/6192) at me, but that I was luckily able to resolve by [disabling hardware acceleration on my FRITZ!Box router](https://github.com/docker/for-win/issues/6192#issuecomment-720084645)!
